flutter/shell/platform/tizen/tizen_view_elementary.h

```diff
@@ -51,6 +51,10 @@ class TizenViewElementary : public TizenView {
   Evas_Object* parent_ = nullptr;
   Evas_Object* container_ = nullptr;
   Evas_Object* image_ = nullptr;
+#ifndef WEARABLE_PROFILE
+  Evas_Object* ctxpopup_ = nullptr;
+  Evas_Object_Event_Cb popup_hide_callback_ = nullptr;
```

Comment:
How about using `evas_object_callbacks_` to store function pointer instead of this?

Reply:
Thank you for the good offer. changed it to store in `evas_object_callbacks_`.